### ISSUE TITLE
Deployment (Automotive) on Google Play requires targetSdk 29

### ISSUE THREAD

**Issue by frankkienl** [android/uamp#396]
See PR: #395 

Google Play Store requires targetSdk 29

**Comment by frankkienl** [comment]
Google Play Store also requires screenshots;
For landscape this is no problem, as the default Android Automotive emulator has the correct screen resolution.
For portrait, is a bit of a hassle. There is no Automotive emulator with the correct resolution.   
But I've found a way to force the emulator to use the required resolution:  
run `adb shell wm size 800x1280` then take the screenshot.
afterwards run `adb shell wm size reset` to revert to the default resolution.

**Comment by frankkienl** [comment]
When developing, and running debug versions of your Automotive app, you probably won't run into this issue; But this is required for deployment on Google Play Store;  

It's required to add the following line to the AndroidManifest.xml, otherwise, the app might not show up in the list of app on Automotive.

```
<application
android:appCategory="audio"
...
/>

<uses-feature
        android:name="android.hardware.wifi"
        android:required="false" />
    <uses-feature
        android:name="android.hardware.screen.portrait"
        android:required="false" />
    <uses-feature
        android:name="android.hardware.screen.landscape"
        android:required="false" />
```

**Comment by nic0lette** [comment]
Fixed in #395 
(Thank you!)
